api-scripts/feesfines/DataPreparation/circ-data-load/checkin-checkout-db-restore.sql

```diff
@@ -0,0 +1,21 @@
+/* To use this script, enter the command:
```

Comment:
I understand why you check in circ-data-load, and one of the reasons is because of the change to this file. If you do this, then a) it introduces two copies of the script, and b) only when this particular version of checkin-checkout-db-restore.sql is used that it can delete the accounts data. I'd suggest to modify https://github.com/folio-org/perf-testing/blob/perf147/workflows-scripts/circulation/check-in-check-out/scripts/checkin-checkout-db-restore.sql and in the README file reference using this script from this location instead of loading and copying the entire set of circ-data-load here.